palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/ExpressionSwitch.input

```diff
@@ -58,4 +58,17 @@ class ExpressionSwitch {
           case 2 -> throw new IllegalArgumentException();
           default -> throw new IllegalStateException();};
   }
+
+
+    public void testWithComments(int y) {
```

Comment:
is this the intended snippet for the `before`? I couldn't see what the difference was manually, and both diff and difft report no differences.

Reply:
I've changed the input to a wrongly formatted switch expression [here](https://github.com/palantir/palantir-java-format/pull/1420/commits/78dc77067779470ce601187fe323b617c8d1c7b9)